### PR TITLE
Improve logging around iwlax2xx-firmware removal

### PR DESCRIPTION
**Pull request by bocekm**
The log message wasn't giving the information whether the package has been removed or not. Actually it was suggesting the package got removed even when in fact it wasn't.

Before (when the package is not installed & not being removed):
```
Checking if java-1.7.0-openjdk is installed.
java-1.7.0-openjdk not installed.
Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware package.

[05/18/2022 18:27:00] TASK - [Convert: Subscription Manager - Download packages] ****************
```

After:
```
Checking if java-1.7.0-openjdk is installed.
java-1.7.0-openjdk not installed.
Checking if the iwl7260-firmware and iwlax2xx-firmware packages are installed.
Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware package.

[05/18/2022 18:27:00] TASK - [Convert: Subscription Manager - Download packages] ****************
```
or
```
Checking if java-1.7.0-openjdk is installed.
java-1.7.0-openjdk not installed.
Checking if the iwl7260-firmware and iwlax2xx-firmware packages are installed.
The iwl7260-firmware and iwlax2xx-firmware packages are not both installed. Nothing to do.

[05/18/2022 18:27:00] TASK - [Convert: Subscription Manager - Download packages] ****************
```

### CODE REVIEW

File: convert2rhel/special_cases.py [hunk 1]
```diff
@@ -50,14 +50,20 @@ def remove_iwlax2xx_firmware():
...
         if iwl7260_firmware and iwlax2xx_firmware:
+            logger.info(
+                "Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware"
+                " package."
+            )
             _, exit_code = run_subprocess(["rpm", "-e", "--nodeps", "iwlax2xx-firmware"])
             if exit_code != 0:
                 logger.error("Unable to remove the package iwlax2xx-firmware.")
+        else:
+            logger.info("Both of the packages are not installed. Nothing to do.")
```
Comment (abadger) [inline convert2rhel/special_cases.py:66]: This is an awkward sentence in English.  Maybe rephrased like this:
```suggestion
            logger.info("The iwlax2xx-firmware and iwlax2xx-firmware packages are not both installed. Nothing to do.")
```

File: convert2rhel/unit_tests/special_cases_test.py [hunk 2]
```diff
@@ -139,21 +139,21 @@ def test_unprotect_shim_x64(mock_is_efi, mock_os_remove, sys_id, is_efi, removal
             False,
             ("output", 0),
             0,
-            "Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware package.",
+            "Both of the packages are not installed. Nothing to do.",
```
Comment (abadger) [inline convert2rhel/unit_tests/special_cases_test.py:142]:
```suggestion
            "The iwlax2xx-firmware and iwlax2xx-firmware packages are not both installed. Nothing to do.",
```

File: convert2rhel/unit_tests/special_cases_test.py [hunk 3]
```diff
@@ -139,21 +139,21 @@ def test_unprotect_shim_x64(mock_is_efi, mock_os_remove, sys_id, is_efi, removal
...
         ),
         (
             False,
             True,
             ("output", 0),
             0,
-            "Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware package.",
+            "Both of the packages are not installed. Nothing to do.",
```
Comment (abadger) [inline convert2rhel/unit_tests/special_cases_test.py:149]:
```suggestion
            "The iwlax2xx-firmware and iwlax2xx-firmware packages are not both installed. Nothing to do.",
```

File: convert2rhel/unit_tests/special_cases_test.py [hunk 4]
```diff
@@ -139,21 +139,21 @@ def test_unprotect_shim_x64(mock_is_efi, mock_os_remove, sys_id, is_efi, removal
...
         ),
         (
             False,
             False,
             ("output", 0),
             0,
-            "Removing the iwlax2xx-firmware package. Its content is provided by the RHEL iwl7260-firmware package.",
+            "Both of the packages are not installed. Nothing to do.",
```
Comment (abadger) [inline convert2rhel/unit_tests/special_cases_test.py:156]:
```suggestion
            "The iwlax2xx-firmware and iwlax2xx-firmware packages are not both installed. Nothing to do.",
```

**Review by abadger — DISMISSED** [review]
Just a suggestion to change the wording.  Otherwise looks good.

File: convert2rhel/special_cases.py (hunk 1 above)
Comment (bocekm) [reply convert2rhel/special_cases.py:66]: Yeah, it seemed a bit odd to me but I couldn't figure out a better phrasing :) Thanks.

**Review by r0x0d — APPROVED** [review]
That's look good to me too.

**Review by abadger — APPROVED** [review]

**Comment by bocekm** [comment]
All integration test failures are unrelated - either unstable RHSM (`HTTP error code 500: Internal Server Error`) or missing RHEL 8.6 repos in the testing Satellite server.

**Comment by r0x0d** [comment]
Merging this, since the integration tests are failing for another reason other than the content of this PR.
